### DISCUSSION TITLE
0.70.0

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#8998] [General]
* Ensure that stdout / stderr relays are created for wslg.exe
* Update Windows SDK to 22621 (Windows 11 22H2)
* Update Microsoft.WSLg to version 1.0.45
    * WSLGd: allow alternate rdp file via .wslgconfig
    * Weston: rdp shell: fix window placement issues
    * Weston: rdp shell: register snap applications on start menu
    * Weston: rdp shell: use predefined constant for g_key_file api
    * Weston: rdp shell: add icon file search retry
    * Weston: xwm: Check size hints in weston_wm_window_is_positioned()
* Update MSRDC to version 1.2.3575

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/0.70.0'>0.70.0</a>.</em>

**Comment by gaia** [comment]
Thanks! Does it take a couple days for it to hit the MS store, Ben?

**Reply by Biswa96** [reply]
It's already in Windows Store now.

**Reply by ianzhuo** [reply]
Is there a store version WSL on windows 10 or just a bug ?
As a windows 10 user, I'm looking forward to see built in systemd support for Win 10 WSL. 
![image](https://user-images.githubusercontent.com/1394735/196232963-5fd90330-55cc-4fea-a2c2-06f45fca21f1.png)
![image](https://user-images.githubusercontent.com/1394735/196232829-5bc2c7dd-0c17-4f9d-97e6-5d097cdb103a.png)

**Reply by Biswa96** [reply]
It is not available for Windows 10 **yet**.

**Reply by wond4** [reply]
Is there a plan to support Windows 10?

**Comment by NotTheDr01ds** [comment]
@wond4 Unlikely.  Newer versions of WSL require functionality only present in Windows 11.

**Comment by NotTheDr01ds** [comment]
@benhillis Any pointers to information on the new `plan9` process that seems to be started by `init` in this release?  Clearly it's related to the 9P sharing, but having a non-init process be started by WSL is a relatively new thing (other than Systemd, of course).  I don't see anything related in the release notes, but 0.70.0 seems to be when it first showed up.

I'm assuming it's a refactoring of some sort to prepare to address an issue (since there's no mention of an issue being fixed by it).  My hope, of course, would be that it's work on the WSL2 9P performance issues.  I'm just curious of those hopes are based on some reality, or if `plan9` is just something more "mundane" 😉.
